backend/sites/src/app/entities/siteHistoryLog.entity.ts

```diff
@@ -21,6 +21,6 @@ export class HistoryLog extends BaseAuditEntity {
   siteId: string;
 
   @Field(() => GraphQLJSON)
-  @Column('jsonb', { name: 'snapshot_data' })
-  content: SaveSiteDetailsDTO;
+  @Column('jsonb', { name: 'data' })
+  content:  Record<string, any>
```

Comment:
What is the purpose for this change?

Reply:
This was due to branch created from a previous commit, but latest dev is merged to this branch and no longer its not part of the pull request.